frontend/app/mixins/graphable.js

```diff
@@ -0,0 +1,94 @@
+import Ember from 'ember';
+
+export default Ember.Mixin.create({
```

Comment:
I like to have all stuff about chart into `components/chart` directory, I'm working on new health-chart components and I'd like to have all stuff to `chart` directory.